csrc/scheduler/normalization_utils.cpp

```diff
@@ -1235,14 +1236,32 @@ void movePersistentBufferToSmem(
       // path of gmem -> smem to reduce temporary register usage. Otherwise, the
       // data path from gmem to shared memory (smem) follows this sequence: gmem
       // -> L1 cache -> register -> smem.
+      // Only supported after device 8.0 and requires vectorized load.
       int hw_major = at::cuda::getCurrentDeviceProperties()->major;
-      if (is_cached_input && hw_major >= 8) {
+      if (rparams->vectorize_inner_reduction && is_cached_input &&
```

Comment:
Why `rparams->vectorize_inner_reduction`? Is it for avoiding bank conflicts?

Reply:
Current main branch tests passed without this predicate is just because we don't have a test using shared memory persistent and can't be vectorized to trigger this error. It was found while working on https://github.com/NVIDIA/Fuser/pull/3175 where all cached input buffers are moved to shared memory and test `CombinedSchedulerTest.LayerNormBackward/dtype___half_batch_216_hidden_1987` will trigger this error.